src/components/app/app.js

```diff
@@ -44,6 +44,23 @@ if (Config.showWalkthroughExamples) {
   tabConfigs.push(gridExampleTab);
 }
 
```

Comment:
Can things like this be moved below or to a separate file? Not a huge deal but I think example stuff shouldn't be the first things someone sees. In some instances like const flyoutExampleTab being near all the other tab configurations makes sense. But individual classes like this seems like they can be moved.

Reply:
The final location of the docs and sample code is still to be determined.  For example, the doc folks want to put these with the rest of the docs (i.e. not in the code).  Having working sample code is valuable though...and the docs folks can't do that. So the sample code might move to its own repo... or get removed altogether. For now, its not really worth moving to separate files or figuring out anything past what it there now... since it may ALL go away anyway.

Reply:
That is fair enough